layouts/partials/search-input.html

```diff
@@ -1,13 +1,9 @@
 {{ if or .Site.Params.gcs_engine_id .Site.Params.algolia_docsearch }}
 <input type="search" class="form-control td-search-input" placeholder="&#xf002; {{ T "ui_search" }}" aria-label="{{ T "ui_search" }}" autocomplete="off">
 {{ else if .Site.Params.offlineSearch }}
-{{ $offlineSearchIndex := resources.Get "json/offline-search-index.json" | resources.ExecuteAsTemplate "offline-search-index.json" . }}
-{{ $offlineSearchLink := $offlineSearchIndex.RelPermalink -}}
-{{ if hugo.IsProduction -}}
-{{/* Use `md5` as finger print hash function to shorten file name to avoid `file name too long` error. */}}
-{{ $offlineSearchIndexFingerprint := $offlineSearchIndex | resources.Fingerprint "md5" }}
-{{ $offlineSearchLink = $offlineSearchIndexFingerprint.RelPermalink -}}
-{{ end -}}
```

Comment:
This change reverts #691. Again, I don't know if this is by mistake or by design, but it seems to be happening a little too often IMHO.

I'm wondering if it is a process issue where folks aren't working from the latest version of docsy and simply cause updates to be lost (reverted) when they commit. It might be worth having a discussion about this, WDYT @LisaFC?

/cc @celestehorgan @nate-double-u

Reply:
FYI, #714 is a proposed fix.